client/lib/domains/resolve-domain-status.tsx

```diff
@@ -185,22 +167,18 @@ export function resolveDomainStatus(
 						listStatusText: status,
 						noticeText: translate(
 							"We noticed that something wasn't updated correctly. Please try {{a}}this setup{{/a}} again.",
-							options
+							{ components: mappingSetupComponents }
 						),
 						listStatusClass: 'alert',
 						listStatusWeight: 1000,
 					};
 				}
 			}
 
-			if ( ( ! isJetpackSite || isSiteAutomatedTransfer ) && ! domain.pointsToWpcom ) {
+			if ( ( ! isJetpackSite || isSiteAutomatedTransfer ) && ! domain.pointsToWpcom && siteSlug ) {
```

Comment:
Is `siteSlug` check necessary?

Reply:
Good catch, it's not 😆 before I refactored some code it was useful, but now it's not.